### PR TITLE
Invite to Slack

### LINKED ISSUE #15: Invite to Slack
If the email address is from IIITV, invite the member to Slack too.

Resource: https://api.slack.com/methods/channels.invite

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #15

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -46,7 +47,10 @@ app.get('/sendmail/:username/:id', (request, response, next) => {
     username: username
   })
 
-  const verificationurl = 'http://' + request.get('host') + '/verify/' + base64
+  var SlackURL = 'https://slack.com/api/users.admin.invite?token=' + slack + '&email=' + id
```
Comment (anshumanv) [inline server.js]: const SlackURL = `https://slack.com/api/users.admin.invite?token=${slack}&email=${id}`

Same below

Comment (aashutoshrathi) [reply server.js]: Yeah, we had that in mind from start, but this doesn't work on glitch for some reason.
That's why the whole file has strings appended with `+`.
:confused:

Comment (anshumanv) [reply server.js]: Didn't know sumimasen 🙏

**Comment by anshumanv** [comment]
✅

**Comment by anshumanv** [comment]
No more issues on Glitch?

**Comment by aashutoshrathi** [comment]
Yeah.
Gave the project hard reset.

**Comment by aashutoshrathi** [comment]
@rossoskull can you just check it on some IDs?

**Comment by rossoskull** [comment]
Sure

**Pull request merged**